CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 ## 3.2.1(unreleased)
+### Notes
+- This release adds support to Oneview Rest API version 800, 1000, and 1200 for C7000 and Synergy to the features Hypervisor Manager.
 
-## Bugs & Enhancements:
-- [#419](https://github.com/HewlettPackard/oneview-chef/issues/419) Dockerfile is failing because of insufficient libraries.
```

Comment:
we don't have to remove anything which is not yet released

Reply:
@soodpr Corrected.